drivers/rtc/Kconfig.stm32_rtc

```diff
@@ -0,0 +1,70 @@
+# Kconfig - STM32 RTC configuration options
+#
+# Copyright (c) 2018 Workaround GmbH
+# Copyright (c) 2018 Allterco Robotics
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config RTC_STM32
+	bool "STM32 RTC Driver"
+	depends on SOC_FAMILY_STM32 && SOC_SERIES_STM32L4X
+	default n
+	select USE_STM32_LL_RTC
+	select USE_STM32_LL_PWR
+	select USE_STM32_LL_RCC
+	select USE_STM32_LL_EXTI
+	select NEWLIB_LIBC
+	help
+	  Build STM32 RTC driver.
+
+choice
```

Comment:
For easier use, add a name to the choice and remove default.
Default value will be the first option listed